eclipse-scout-core/src/style/mixins.less

```diff
@@ -633,4 +633,14 @@
       }
     }
   }
+
+  /**
+   * Adds a rule for a css class with the name @{edge}-resize that defines the cursor with the same name, e.g. resize-w.
+   */
+  .resize(@edge) {
+    .@{edge}-resize,
+    .@{edge}-resize * { /* NOSONAR (universal selector is okay here) */
+      cursor: ~'@{edge}-resize !important'; /* NOSONAR !important is necessary to override any defined cursor on underlying components */
```

Comment:
You probably tested this, but why is the value a string? Isn't this usually done to prevent LESS from converting anything inside? But @{edge} should be replaced with the actual value.

Reply:
This is the only way I got it to compile, the output looks correct, the single quotes will be gone and the variable replaced.